docs/api/en/math/Box3.html

```diff
@@ -12,11 +12,37 @@ <h1>[name]</h1>
 
 		<p class="desc">
 			Represents a box or cube in 3D space. The main purpose of this is to represent
-			the [link:https://en.wikipedia.org/wiki/Minimum_bounding_box Minimum Bounding Boxes]
-			for objects.
+			the world-axis-aligned bounding boxes for objects. 
 		</p>
 
+		
+		<h2>Example</h2>
 
+		<code>
+		// Creating the object whose bounding box we want to compute
+		var sphereGeom = new THREE.SphereGeometry();
+		var sphereObject = new THREE.Mesh( 
+			sphereGeom, 
+			new THREE.MeshBasicMaterial( 0xff0000 ) 
+		);
+
+		// Creating the actual bounding box with Box3
+		sphereObject.geometry.computeBoundingBox();
+		var BBox = new THREE.Box3(
+			sphereObject.geometry.boundingBox.min,
+  			sphereObject.geometry.boundingBox.max
+		);
+
+		// Moving the sphere
+		sphereObject.position.set( 7, 7, 7 );
+
+		// Updating the bounding box
+		// (to repeat each time the object is moved, rotated or scaled) 
+		BBox.copy( sphereObject.geometry.boundingBox ).applyMatrix4( sphereObject.matrixWorld );
```

Comment:
This code is wrong since you did not update the world matrix after setting the position.

Reply:
Oh sorry, for some reason my tests worked even without updating it so I forgot it. I'm gonna fix it asap.